ray-operator/config/rbac/role.yaml

```diff
@@ -75,6 +107,32 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - rayservice.io
+  resources:
+  - rayservices
+  verbs:
+  - create
```

Comment:
Hmm, just curious -- Why does the operator need permissions to create and delete RayServices?
(And why did the operator need permissions to create and edit RayClusters before the introduction of RayServices?)

Isn't the controller for a given resource is supposed to only read from spec and only update status?

Reply:
I think every controller need permissions for its own CR. And RayService will maintain RayCluster, so I give RayService the RayCluster permission too.

Reply:
Right -- the RayService controller needs to be able to create and delete RayClusters.

It's probably fine as is, but creating and deleting RayServices is not a function of the RayService controller -- users create/delete and the RayService controller responds. The RayService controller probably only needs write access to the status subresource.

Reply:
(And read access to the main resource.)

Reply:
RayService and RayServices rabc are auto generated. I think RayService controller is the owner of RayService(s) CRD, so better to let it have all access.

Reply:
Ok, sure.